.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Run tests
+on: push
```

Comment:
The circleCI workflow runs on every branch except `master`, but I am not sure if it should really be excluded from this simple test workflow.

Reply:
Testing on master would be too late and should be done before.
There is no harm in executing tests on master again.